modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ModalAddObjectLayoutTab.tsx

```diff
@@ -261,6 +261,7 @@ export function ModalAddObjectLayoutTab({
 								'there-are-no-relationship-for-this-object'
 							)}
 							error={errors.objectRelationshipId}
+							id="modal-add-object-layout-tab"
```

Comment:
Hi @Artartur. Please, rename the id to `modalAddObjectLayoutTab`.

Reply:
Done!